parts/JavaConfig-JSP/projectName-web/src/main/java/xxxxxx/yyyyyy/zzzzzz/config/web/SpringSecurityConfig.java

```diff
@@ -31,7 +31,7 @@ public class SpringSecurityConfig {
      */
     @Bean
     public WebSecurityCustomizer webSecurityCustomizer() {
-        return web -> web.ignoring().requestMatchers("/resources/**");
+        return web -> web.ignoring().requestMatchers(new AntPathRequestMatcher("/resources/**"));
```

Comment:
@takahashihrzg @btkatoufm 
I think it's better to use static imported `AntPathRequestMatcher#antMatcher(String)`. HDYT？

https://docs.spring.io/spring-security/reference/servlet/authorization/authorize-http-requests.html#_request_matchers